flink-dist/src/main/flink-bin/bin/config.sh

```diff
@@ -186,6 +186,9 @@ KEY_ENV_JAVA_OPTS_TM="env.java.opts.taskmanager"
 KEY_ENV_JAVA_OPTS_HS="env.java.opts.historyserver"
 KEY_ENV_JAVA_OPTS_CLI="env.java.opts.client"
 KEY_ENV_JAVA_OPTS_SQL_GATEWAY="env.java.opts.sql-gateway"
+KEY_ENV_JAVA_DEFAULT_OPTS="env.java.default-opts.all"
```

Comment:
`"env.java.opts.all"` are supported in `pyflink_gateway_server.py`, `"env.java.default-opts.all"` should be considered there as well.